consensus/entry_point_selector/tests/test_entry_point_selector.c

```diff
@@ -7,10 +7,65 @@
 
 #include <unity/unity.h>
 
+#include "common/helpers/digest.h"
+#include "common/model/tests/defs.h"
 #include "consensus/entry_point_selector/entry_point_selector.h"
+#include "consensus/milestone/milestone.h"
+#include "consensus/tangle/tangle.h"
+#include "utils/files.h"
+
+static entry_point_selector_t eps;
+static tangle_t tangle;
+static milestone_t milestone;
+
+static char* test_db_path = "consensus/entry_point_selector/tests/test.db";
```

Comment:
I think we have these in too many places, can we create functions in `consensus/test_definitions` ?
`get_test_db_path` + `get_src_db_path` so it will also allow for debugging (the function will append  test.db/ciri.db to CWD)
And also, test_setup/test_cleanup